### PR TITLE
A few more dbus api documentation updates

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: man/org.freedesktop.login1.xml [hunk 1]
```diff
@@ -606,8 +606,8 @@ node /org/freedesktop/login1/seat/seat0 {
       encoded in a structure consisting of the ID and the object path.</para>
 
       <para>The <varname>IdleHint</varname>, <varname>IdleSinceHint</varname>, and
-      <varname>IdleSinceHint</varname> properties encode the idle state, similar to the one exposed on the
-      Manager object, but specific for this seat.</para>
+      <varname>IdleSinceHintMonotonic</varname> properties encode the idle state, similar to the one exposed
```
Comment (daandemeyer) [inline man/org.freedesktop.login1.xml]:
```suggestion
      <varname>IdleSinceHintMonotonic</varname> properties encode the idle state, similar to the ones exposed
```

Comment (keszybz) [reply man/org.freedesktop.login1.xml]: Fixed, thanks.

**Comment by keszybz** [comment]
Rebased.

**Comment by keszybz** [comment]
...and one more commit added.

**Comment by keszybz** [comment]
Rebased again to fix a conflict.

**Pull request merged**